framework/core/src/Foundation/Application.php

```diff
@@ -140,10 +140,10 @@ protected function registerBaseBindings()
          * Use container inside flarum instead.
          */
         $this->container->instance('app', $this->container);
-        $this->container->alias('app', \Illluminate\Container\Container::class);
+        $this->container->alias('app', \Illuminate\Container\Container::class);
```

Comment:
These probably should be imported though, unless I missed another explanation of why not?

Reply:
Fet out of scope, there are so many instances like these, and this PR does so much already, I'd rather we do anything else that isn't necessarily related to setting up phpstan in other PRs.

Reply:
Good point, let's aim to get this merged shortly after bundling the release so that we can build on it going forward.